### PR TITLE
Add default route key name for implicit binding

### PR DESCRIPTION
**Pull request by joelcuevas**

### CODE REVIEW

File: src/EloquentHashids.php [hunk 1]
```diff
@@ -51,4 +51,12 @@ public static function getHashidEncodingValue(Model $model)
   {
     return $model->getKey();
   }
+  
+  /**
+   * @return string
+   */
+  public function getRouteKeyName()
+  {
+      return 'uid';
```
Comment (mikerockett) [inline src/EloquentHashids.php:60]: Surely this should return the result of the hashid column method? That way, it only needs to be set once.

Comment (joelcuevas) [reply src/EloquentHashids.php:60]: @mikerockett sorry, didn't understand your question, what do you mean?

Comment (mikerockett) [reply src/EloquentHashids.php:60]: What I'm saying is that, under normal circumstances, the route name should be the same as the column name. As such, the `getRouteKeyName` function should return the name of the column (through `getHashidColumn`), which could then be overriden if required.

Comment (joelcuevas) [reply src/EloquentHashids.php:60]: Ohhhh! Yes, of course. I'll change it on the PR.

Comment (joelcuevas) [reply src/EloquentHashids.php:60]: Although, I just remembered.... I don't know if there is a way to access the current model. The `getHashidColumn` method expects it as parameter (don't know why).

Maybe just passing `null` should do the job. What do you think?

Comment (naabster) [reply src/EloquentHashids.php:60]: thanks - I will look at it this week. It's a bit much with all this christmas right now :)

Comment (joelcuevas) [reply src/EloquentHashids.php:60]: Great! Happy holidays! ;)

**Pull request closed without merging**